ui/app/routes/jobs/index.js

```diff
@@ -32,7 +43,12 @@ export default class IndexRoute extends Route.extend(
     controller.set('namespacesWatch', this.watchNamespaces.perform());
     controller.set(
       'modelWatch',
-      this.watchJobs.perform({ namespace: controller.qpNamesapce })
+      this.watchJobs.perform({
+        namespace: controller.qpNamespace,
+        per_page: controller.pageSize,
+        filter: `ParentID is empty`,
+        next_token: controller.nextToken,
+      })
     );
   }
```

Comment:
We have no easy path forward on either option.

Datacenter is just metadata. We have no way to get a full list of this.
Prefix is also a big challenge because that's a client-side computation.